includes/class-wc-payments.php

```diff
@@ -480,4 +481,15 @@ public static function get_file_version( $file ) {
 	public static function get_gateway() {
 		return self::$gateway;
 	}
+
+	/**
+	 * Registers the payment method with the blocks registry.
+	 *
+	 * @param Automattic\WooCommerce\Blocks\Payments\PaymentMethodRegistry $payment_method_registry The registry.
+	 */
+	public static function register_checkout_gateway( $payment_method_registry ) {
+		require_once dirname( __FILE__ ) . '/class-wc-payments-payment-method.php';
```

Comment:
Similar to comment above, please consider including "blocks" in the name of this file

Reply:
See https://github.com/Automattic/woocommerce-payments/pull/723#discussion_r494209561